metricflow/test/integration/test_cases/itest_conversion_metrics.yaml

```diff
@@ -0,0 +1,44 @@
+---
+integration_test:
+  name: conversion_rate_metric
+  description: Query a conversion metric that calculates the conversion rate
+  model: SIMPLE_MODEL
+  metrics: ["visit_buy_conversion"]
+  group_bys: ["metric_time"]
```

Comment:
We also want a test where we group by referrer_id. If that's a follow-up PR it's fine, just making sure we don't forget it.